src/routes/constraints/edit/[id]/+page.svelte

```diff
@@ -35,24 +40,32 @@
   let referenceModelId: number | null = null;
 
   $: $constraintMetadataId = data.initialConstraint.id;
-  $: if ($constraintMetadata != null && $constraintMetadata.id === $constraintMetadataId) {
-    constraintDefinition = $constraintMetadata.versions.find(
-      ({ revision }) => revision === constraintRevision,
-    ) as ConstraintMetadataVersionDefinition;
-    if (constraintDefinition != null) {
-      constraintDefinitionAuthor = constraintDefinition?.author;
-      constraintDefinitionCode = constraintDefinition?.definition;
-      constraintDefinitionTags = constraintDefinition?.tags.map(({ tag }) => tag);
-    }
+  $: (async () => {
```

Comment:
Did not know this sort of pattern was possible in svelte! Any reason not to just put this logic in a function and then pass it whatever is needed? Not sure if that is any better though, just curious.

Reply:
That's a great point... :facepalm: